bundles/org.openhab.binding.jellyfin/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,180 @@
+				<label>Port</label>
+				<description>Port of the server</description>
+				<default>8096</default>
+			</parameter>
+			<parameter name="ssl" type="boolean" required="true">
+				<label>SSL</label>
+				<description>Connect through https</description>
+				<default>false</default>
+			</parameter>
+			<parameter name="refreshSeconds" type="integer" min="10" max="300" required="true">
+				<label>Refresh Seconds</label>
+				<description>Interval to pull devices state from the server</description>
+				<default>30</default>
+			</parameter>
+			<parameter name="clientActiveWithInSeconds" type="integer" min="0" max="950" required="true">
+				<label>Client Active With In Seconds</label>
+				<description>Amount off seconds allowed since the last client activity to assert it's online (0 disabled)</description>
+				<default>0</default>
+			</parameter>
+			<parameter name="userId" type="text">
+				<label>User ID</label>
+				<description>The user id</description>
+			</parameter>
+			<parameter name="token" type="text">
+				<label>Access Token</label>
+				<description>The user access token</description>
+			</parameter>
+		</config-description>
+	</bridge-type>
+	<!-- Sample Thing Type -->
+	<thing-type id="client">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="server"/>
+		</supported-bridge-type-refs>
+
+		<label>Jellyfin Client</label>
+		<description>Represents a running Jellyfin client connected to a server</description>
+
+		<channels>
+			<channel id="play-state-command" typeId="play-state-command-channel"/>
+			<channel id="send-notification" typeId="send-notification-channel"/>
+			<channel id="media-control" typeId="system.media-control"/>
+			<channel id="playing-item-name" typeId="playing-item-name-channel"/>
+			<channel id="playing-item-series-name" typeId="playing-item-series-name-channel"/>
+			<channel id="playing-item-season-name" typeId="playing-item-season-name-channel"/>
+			<channel id="playing-item-season" typeId="playing-item-season-channel"/>
+			<channel id="playing-item-episode" typeId="playing-item-episode-channel"/>
+			<channel id="playing-item-genders" typeId="playing-item-genders-channel"/>
+			<channel id="playing-item-type" typeId="playing-item-type-channel"/>
+			<channel id="playing-item-percentage" typeId="playing-item-percentage-channel"/>
+			<channel id="playing-item-second" typeId="playing-item-second-channel"/>
+			<channel id="playing-item-total-seconds" typeId="playing-item-total-seconds-channel"/>
+			<channel id="play-by-terms" typeId="play-by-terms-channel"/>
+			<channel id="play-next-by-terms" typeId="play-next-by-terms-channel"/>
+			<channel id="play-last-by-terms" typeId="play-last-by-terms-channel"/>
+			<channel id="browse-by-terms" typeId="browse-by-terms-channel"/>
+		</channels>
+
+		<config-description>
+		</config-description>
+	</thing-type>
+
+	<!-- Client Channels -->
+	<channel-type id="play-state-command-channel">
+		<item-type>String</item-type>
```

Comment:
Why are there two Channels with different interfaces to control the media?

Reply:
I think this one allow a more grained control, the other one is the standard media controller channel. I can remove this one if you want, or maybe mark this one as advanced?

Reply:
I couldn't find any function of this Channel that can't also be used with the ordinary Player Item. If this Channel has superior functionality over the Player Item, you can keep it.